ontologies/WIP/sierratransformable

```diff
@@ -459,38 +459,20 @@
    *
 
 
-   * Populate wwork:publicationDate. Rules:
-   * 
-   * 1. For bibliographic records where "260" is populated
-   * 2. Platform "publicationDate" "label" populate with 260 subfield "c"
-   * 3. Type as "Period"
-   * 4. Also all bibliographic records use "publishYear"
-   * 5. Platform "year" populate from "publishYear"
-   *
-   *
-   * Example: 
-   *   "publicationDate": {
-   *     "label": "23 May 1932",
-   *     "type": "Period" 
-   *  },
-   * 
-   *
-   * Example of how we might pull out year or date when we know it in the future:
-   * 
-   *   "publicationDate": {
-   *     "label": "May 23 1932",
-   *     "type": "Period" ,
-   *     "year": "1932" ,
-   *     "date" : "05-23-1932"
-   *  },
-   */
+    // @@AWLC
```

Comment:
should this still be here?

Reply:
Mmm. This is a file of notes, the @@AWLC is just a note of who wrote them. I’ll leave it for now, as it’s useful context.